dspace-server-webapp/src/test/java/org/dspace/app/rest/CommunityCollectionParentIT.java

```diff
@@ -0,0 +1,311 @@
+
+        String content = mvcResult.getResponse().getContentAsString();
+        Map<String, Object> map = mapper.readValue(content, Map.class);
+        String communityUuidString = String.valueOf(map.get("uuid"));
+        String communityName = String.valueOf(map.get("name"));
+
+        mvcResult = getClient(token).perform(get("/api/core/communities/" + communityAA.getID())).andReturn();
+        content = mvcResult.getResponse().getContentAsString();
+        map = mapper.readValue(content, Map.class);
+        String actualCommunityName = String.valueOf(map.get("name"));
+
+        assertThat(communityName, equalTo(actualCommunityName));
+        assertThat(communityUuidString, equalTo(String.valueOf(communityAA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityAB.getID())));
+
+    }
+
+    @Test
+    public void col2ParentCommunityTest() throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        String token = getAuthToken(admin.getEmail(), password);
+        MvcResult mvcResult = getClient(token)
+            .perform(get("/api/core/collections/" + col2.getID() + "/parentCommunity")).andReturn();
+
+        String content = mvcResult.getResponse().getContentAsString();
+        Map<String, Object> map = mapper.readValue(content, Map.class);
+        String communityUuidString = String.valueOf(map.get("uuid"));
+        String communityName = String.valueOf(map.get("name"));
+
+        mvcResult = getClient(token).perform(get("/api/core/communities/" + communityAA.getID())).andReturn();
+        content = mvcResult.getResponse().getContentAsString();
+        map = mapper.readValue(content, Map.class);
+        String actualCommunityName = String.valueOf(map.get("name"));
+
+        assertThat(communityName, equalTo(actualCommunityName));
+        assertThat(communityUuidString, equalTo(String.valueOf(communityAA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityA.getID())));
+        assertThat(communityUuidString, not(String.valueOf(communityAB.getID())));
+
+    }
+
+    @Test
+    public void col3ParentCommunityTest() throws Exception {
+
+        ObjectMapper mapper = new ObjectMapper();
+        String token = getAuthToken(admin.getEmail(), password);
+        MvcResult mvcResult = getClient(token)
+            .perform(get("/api/core/collections/" + col3.getID() + "/parentCommunity")).andReturn();
+
+        String content = mvcResult.getResponse().getContentAsString();
+        Map<String, Object> map = mapper.readValue(content, Map.class);
+        String communityUuidString = String.valueOf(map.get("uuid"));
+        String communityName = String.valueOf(map.get("name"));
+
+        mvcResult = getClient(token).perform(get("/api/core/communities/" + communityAB.getID())).andReturn();
```

Comment:
we can avoid to retrieve this value from the REST, the findOne endpoint has already test in the specific IT class. Doing it again here will only slow our test